Makefile

```diff
@@ -0,0 +1,74 @@
+# Go parameters
+GOCMD	= go
+GOBUILD	= $(GOCMD) build
+GOCLEAN	= $(GOCMD) clean
+GOTEST	= $(GOCMD) test
+GOTOOL	= $(GOCMD) tool
+GOGET	= $(GOCMD) get
+GOMOD	= $(GOCMD) mod
+GOINST	= $(GOCMD) install
+GORUN	= ${GOCMD} run
+
+BUILD_DIR		:= build
+TEST_BUILD_DIR  := build/test
+BINARY_NAME		:= nginx-agent
+BINARY			:= $(BUILD_DIR)/$(BINARY_NAME)
+PROJECT_DIR		= cmd/agent
+PROJECT_FILE	= main.go
+IMPORT_MAPPING 	:= ../common/common.yaml:github.com/nginx/agent/v3/internal/api/http/common
+
+help: ## Show help message
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\033[36m\033[0m\n"} /^[$$()% 0-9a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-24s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+clean: ## Remove build directory
+	@rm -rf ${BUILD_DIR}
+	@echo "🌀 Cleaning Done"
+
+no-local-changes:
+	git diff --quiet || { echo "Dependency changes detected. Please commit these before pushing." >&2; exit 1; }
+
+build: ## Build agent executable
+	@mkdir -p $@
+	@$(GOBUILD) -o $(BINARY) $(PROJECT_DIR)/${PROJECT_FILE}
+	@echo "📦 Build Done"
+
+lint: ## Run linter
+	GOWORK=off go vet ./...
+	GOWORK=off go run github.com/golangci/golangci-lint/cmd/golangci-lint run -c ./scripts/.golangci.yml
+
+format: ## Format code
+	@$(GORUN) mvdan.cc/gofumpt -l -w .
+	@echo "🏯 Format Done"
+
+install-tools: ## Install dependencies in tools.go using vendored version see https://www.jvt.me/posts/2023/06/19/go-install-from-mod/
+	@echo "Installing Tools"
+	@grep _ ./tools/tools.go | awk '{print $$2}' | xargs -tI % env GOBIN=$$(git rev-parse --show-toplevel)/bin GOWORK=off go get %
```

Comment:
I'm assuming that this line gets the vendored version, which we no longer have, so we can delete it.

Reply:
I have another commit coming soon that will change how we handle installing tools so that we install a specific version each time